api/prisma/schema.prisma

```diff
@@ -0,0 +1,22 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Game {
+  id                Int     @id @default(autoincrement())
+  board             Json?
+  currentPlayer     Int?
+  players           Json?
+  directionSelected String?
+  marbleClicked     Json?
+  hasWinner         Boolean @default(false)
+  started           Boolean @default(false)
```

Comment:
Can you have `hasWinner` at `true` and `started` at `true` ? Is it relevant ? Maybe you can use string status instead.
not_started | started | finished it will simplify your code IMHO

Feel free to not do this if you'r in late or do it in another PR. It's also a good practice to avoid too much boolean on an entity and prefer using a string status field.

Reply:
it would implies may changes, but we have to modify some logic, it will be done at this moment. 
It's clearly more logical.